### PR TITLE
Docs: Update index.md

### PR DESCRIPTION
**Pull request by oddlittlebird**
Minor edit, troubleshooting things with Aparajita. If you aren't her, please do not approve this.

### CODE REVIEW

File: docs/sources/features/explore/index.md [hunk 1]
```diff
@@ -31,11 +31,11 @@ There is an Explore icon on the menu bar to the left. This opens an empty Explor
 
 {{< docs-imagebox img="/img/docs/v65/explore_menu.png" class="docs-image--no-shadow" caption="Screenshot of the new Explore Icon" >}}
 
-If you want to start with an existing query in a panel then choose the Explore option from the Panel menu. This opens an Explore tab with the query from the panel and allows you to tweak or iterate in the query outside of your dashboard.
+If you want to start with an existing query in a panel, choose the Explore option from the Panel menu. This opens an Explore tab with the query from the panel and allows you to tweak or iterate in the query outside of your dashboard.
```
Comment (oddlittlebird) [inline docs/sources/features/explore/index.md:34]: Type a  comment

Comment (oddlittlebird) [inline docs/sources/features/explore/index.md:34]:
```suggestion
If you want to start with an existing query in a panel, and then choose the Explore option from the Panel menu. This opens an Explore tab with the query from the panel and allows you to tweak or iterate in the query outside of your dashboard.
```

**Review by oddlittlebird — COMMENTED** [review]
LGTM! Just a few suggestions

**Review by achatterjee-grafana — APPROVED** [review]
Looks good.

**Pull request merged**